src/views/StoryViewer.vue

```diff
@@ -6,7 +6,8 @@
         <div v-else-if="currentFrame.id > n" class="loaded"></div>
       </div>
     </div>
-    <p v-if="currentFrame.text">{{ currentFrame.text }}</p>
+    <div class="story-text" v-if="currentFrame.text">{{ currentFrame.text }}</div>
+    <div class="restart-btn" v-if="currentFrame.id === storyLength" @click="selectFrame(1)">Start again</div>
```

Comment:
Please use $i18n() for the message

Reply:
You can created a `computed` called `lastFrame` or something to avoid complex expressions in the template.